### DISCUSSION TITLE
Maui Preview 6 .net 8 latest visual studio Hot restart cannot build it!

### DISCUSSION THREAD

**Discussion by TedMobile** [dotnet/maui#16439] [General]
Hi
wondering if anyone has managed to get iOS working against a device using windows as host.

Regardless what I do with my provisioning profiles I get below - no issues with .net 7 any suggestions?

Severity	Code	Description	Project	File	Line	Suppression State
Error		Project bundle identifier 'com.companyname.mauiapp2' does not match specified provisioning profile 'VS: WildCard Development'. Please enable Automatic Provisioning from the iOS Bundle Signing page.	MauiApp2	C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.4.8646-net8-p6\tools\msbuild\iOS\Xamarin.iOS.HotRestart.targets	36

**Comment by nzohrab** [comment]
Hey Just wondering whether you ended up solving this? I'm having the exact same issue.
